remote-api/src/main/java/org/alfresco/rest/api/impl/AuditImpl.java

```diff
@@ -913,7 +913,7 @@ public int getAuditEntriesCountByAppAndProperties(AuditService.AuditApplication
         final String applicationName = auditApplication.getKey().substring(1);
```

Comment:
Maybe we can change this line to:

`final String applicationName = auditApplication.getName();`

Reply:
Changes are done as per the comment.